Integrations/build.gradle

```diff
@@ -102,16 +77,29 @@ Docker.registerDockerImage(project, 'buildDeephavenPython') {
     // deephaven2.whl
     def deephaven2Wheel = project(':deephaven2-wheel').tasks.getByName('buildWheel')
 
-    // Dockerfile
-    def dockerfile = deephavenPythonDockerfile.get()
+    from (deephavenJpyWheel.outputs.files) {
+        into 'deephaven-jpy-wheel'
+    }
+    from (deephavenWheel.outputs.files) {
+        into 'deephaven-wheel'
+    }
+    from (deephaven2Wheel.outputs.files) {
+        into 'deephaven2-wheel'
+    }
+
+    from 'src/main/docker'
```

Comment:
why not embed the Dockerfile as a task, rather than an "external" build file to show how to use the wheels mentioned above?

Also, along with the "one big copy" comment, consider making the three `from`s use one common `into`?

Reply:
I don't think of it so much as an "external" build file. When we don't need build-level parameterization of a dockerfile, I think a textual Dockerfile is easier to grok and modify.

Reply:
Perhaps - I find it easier to all have in one place, rather than knowing when to look at py, groovy, dockerfile, .gradle, .sh, etc to find configuration, consolidate the sh into Dockerfile, the Dockerfile into gradle. 

Plus if we have a centralized tagging system (so we can produce arm images correctly) we are going to have to modify all these images to have the right tag for in-build. For example `FROM dh/java-and-python:local-build-arm64`, or else we have to do a docker+gradle clean on CI to get it to start the next platform, etc. (or pick another option which keeps gradle in the dark about what kinds of images it is producing).

Reply:
In a follow-up PR, I'm adding a "requirements.txt" file - now, we *could* consolidate the requirements file into a RUN/sh install command instead (no file). But I think as a text file it's easier to maintain, and in perspective lives right next to the Dockerfile.